components/HomeScreen/Title/Title.utils.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * @callback setGlobeOrientation
+ * @param {number}
+ */
+
+/**
+ * yieldRotateGlobe.
+ *
+ * @param {number} globeOrientation
+ * @param {function} setGlobeOrientation
+ */
+export function yieldRotateGlobe(globeOrientation, setGlobeOrientation) {
```

Comment:
for this pattern where we extract a function to a util that returns a function to use on the jsx layer we typically use the naming convention `generate` followed by whatever the function it returns is. So in this case we'd call this `generateYieldRotateGlobe`. I also see that you are using it for your onPress so maybe a name like `generateRotateGlobeOnPress` would be appropriate.

Reply:
`generate` is the name that I have used in the past, but this caused a bit of confusion with others because generator functions are already a thing in javascript. When I was describing this pattern to my previous team I called these "generator functions" or "using the generator pattern" and their previous knowledge made it harder for them to learn this pattern. `yield` seems like a good alternative to me. What do you think?

Reply:
I'm sorry I didn't see this comment till today. I think this is another case I'd like to have a larger discussion with the team. Thanks for bringing up the concern. If I don't remember to bring this up can you remind me?